server/cache/dirtools/BUILD

```diff
@@ -1,6 +1,7 @@
 load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
 
 package(default_visibility = [
+    "//cli/download:__pkg__",
     "//cli/remotebazel:__pkg__",
     "//enterprise:__subpackages__",
 ])
```

Comment:
curious, why do we have custom visibility here?

Reply:
This package used to be under //enterprise. It was moved out in 41298bb20a7a166c559176ec37580de2a54b2d45 but the visibility was not updated.

I think we can just remove this entire `package()` call.